src/App.tsx

```diff
@@ -15,48 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
```

Comment:
```suggestion
enum SortType {
  Alphabet = 'alphabet',
  Length = 'length',
```
Enum properties should be capitalized
https://www.w3schools.com/typescript/typescript_enums.php